mir-module/src/main/resources/xsl/mir-utils.xsl

```diff
@@ -101,9 +102,16 @@
                 <xsl:value-of select="concat($baseURL,'images/svg_icons/download_video.svg')"/>
             </xsl:when>
             <xsl:otherwise>
-                <xsl:value-of select="concat($baseURL,'images/svg_icons/download_default.svg')"/>
+                <xsl:choose>
+                    <xsl:when test="string-length($derivateMaindoc) &gt; 0 and string-length($returnId) &gt; 0">
+                        <xsl:variable name="ext" select="substring-after($derivateMaindoc, '.')"/>
```

Comment:
Method `substring-after` is problematic when the filename contains multiple dots, like [`mods2marc-3.7map-base-v.1.0-master.xlsx`](https://www.loc.gov/standards/mods/v3/source/mods2marc-3.7map-base-v.1.0-master.xlsx), which would result in the "extension" `.7map-base-v.1.0-master.xlsx`.

**Suggestion:** Could maybe reuse something existing to obtain the extension. For example, in `mir-video.js.xsl`, `org.apache.commons.io.FilenameUtils#getExtension(String)` is used. Could even be be called without a Xalan-extension:

```
<xsl:variable name="ext" select="document(concat('callJava:org.apache.commons.io.FilenameUtils:getExtension:',$derivateMaindoc))"/>